src/NHibernate/Dialect/MySQL55Dialect.cs

```diff
@@ -8,6 +9,8 @@ public class MySQL55Dialect : MySQL5Dialect
 		public MySQL55Dialect()
 		{
 			RegisterColumnType(DbType.Guid, "CHAR(36)");
+
+			RegisterFunction("strguid", new SQLFunctionTemplate(NHibernateUtil.String, "?1"));
```

Comment:
This overrides the rather complex fix in `MySQL5Dialect`. Currently the CI test with `MySQL5Dialect`. Is this override tested somewhere? Locally?

Reply:
No, it isn't. But in an excuse I would say that in this dialect GUIDs are stored as CHAR(36) in a normal guid format.